README.md

```diff
@@ -653,4 +654,108 @@ amplify push
 
 ---
 
-To learn more about uploading images in a GraphQL application with Amplify and AppSync, check out my tutorial [How to Manage Image & File Uploads & Downloads with AWS AppSync & AWS Amplify](https://dev.to/dabit3/graphql-tutorial-how-to-manage-image-file-uploads-downloads-with-aws-appsync-aws-amplify-hga)
\ No newline at end of file
+To learn more about uploading images in a GraphQL application with Amplify and AppSync, check out my tutorial [How to Manage Image & File Uploads & Downloads with AWS AppSync & AWS Amplify](https://dev.to/dabit3/graphql-tutorial-how-to-manage-image-file-uploads-downloads-with-aws-appsync-aws-amplify-hga)
+
+## Giphy Clone
+
+To deploy this app, use the following steps:
+
+1. Create the Amplify project in your app
+
+```sh
+amplify init
+```
+
+2. Add a lambda function
+
+```sh
+amplify add function
+```
+
+3. Enter the following when prompted
+   ![lambdafunction](screenshots/giphy-clone/create-lambda-function.png)
+
+4. From the root of your project, change into your function's source directory
+
+```sh
+cd amplify/backend/function/giphyfunction/src
+```
+
+5. Install [axios](https://www.npmjs.com/package/axios) and [dotenv](https://www.npmjs.com/package/dotenv)
+
+```sh
+npm install axios && npm install -D dotenv
+```
+
+6. Update the contents of your functions `index.js` file
+
+```js
+const axios = require("axios");
+const dotenv = require("dotenv");
+dotenv.config();
+
+exports.handler = (event, _, callback) => {
+  const { GIPHY_API_KEY } = process.env;
+  let apiUrl = `https://api.giphy.com/v1/gifs/search?api_key=${GIPHY_API_KEY}&q=hello`;
+
+  if (event.arguments) {
+    const { searchTerm = "hi", limit = 25 } = event.arguments;
+    apiUrl = `https://api.giphy.com/v1/gifs/search?api_key=${GIPHY_API_KEY}&q=${searchTerm}&limit=${limit}`;
+  }
+
+  axios
+    .get(apiUrl)
+    .then(response => callback(null, response.data.data))
+    .catch(err => callback(err));
+};
+
+```
+
+7. Login to [Giphy's Developer portal](https://developers.giphy.com/dashboard/) and create an app to get an API Key.
+   
+8. Back in the function's `src` directory, create a `.env` file and add in the API Key that was obtained in the previous step as a secret. 
+
+```sh
+GIPHY_API_KEY=<YOUR_API_KEY>
+```
+
+**Your .env file should NOT be committed to Github**
+
+9. Add the GraphQL API
```

Comment:
Looking at it....would you like me to tell the user to go back to the root of their project before adding the api?

Reply:
I went ahead and changed it to give some clarity.